.github/workflows/shellcheck.yaml

```diff
@@ -0,0 +1,28 @@
+name: ShellCheck
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - release-*
+  pull_request:
+    branches:
+      - master
+      - release-*
+
+jobs:
+  shellcheck:
+    name: Shellcheck
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run ShellCheck
+      uses: ludeeus/action-shellcheck@master
+      with:
+        severity: warning
+        check_together: 'yes'
+        additional_files: 'run finish'
```

Comment:
Where are the `run` and `finish` files?

Reply:
I see `build/run`. But also `build/reset` and `build/sed-in-place`.

maybe this should be `additional_files: build/run build/reset build/sed-in-place`